uber/sep_commands.py

```diff
@@ -12,6 +12,17 @@ def print_config():
 
 
 @entry_point
+def resave_all_attendees_and_groups():
+    Session.initialize_db(modify_tables=True)
+    with Session() as session:
+        print("re-saving all attendees....")
+        [a.presave_adjustments() for a in session.query(Attendee).all()]
+        print("re-saving all groups....")
+        [g.presave_adjustments() for g in session.query(Group).all()]
+        print("Done!")
```

Comment:
Minor note for the future: this ``print('Done!')`` should really be outside the ``with`` block, because the way we have it, you're printing Done *before* we commit.  If something went wrong with the commit we'd get the Done message and then an error, and the user would probably be confused about what was done and what wasn't.